cypress/fixtures/checkAnswersRows.json

```diff
@@ -53,11 +53,7 @@
       "value": "No"
     },
     {
-      "key": "Did you pay home energy costs on a reserve?",
-      "value": "No"
```

Comment:
I would think we still want to keep this.

Reply:
It is now, 'Did you live on a reserve?'

Reply:
should they both be in there?